data-pipeline/ep-core/src/main/java/org/ekstep/ep/samza/task/BaseSamzaTask.java

```diff
@@ -22,14 +26,18 @@ public BaseSamzaTask() {
 	public void initTask(Config config, JobMetrics metrics) {
 		this.metrics = metrics;
 		this.metricsTopic = config.get("output.metrics.topic.name", "telemetry.pipeline_metrics");
+		this.prometheusMetricsTopic = config.get("output.prometheus.metrics.topic.name", "telemetry.metrics");
+		this.metricsList = config.getList("pipeline.metrics.list");
```

Comment:
Can we have a default for the pipeline.metrics.list? If it is not provided, it should take all values.

Reply:
Yes. We can have a default list to fall back upon.